### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by moprescu**
Reflect code changes, update references and publication sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ To install from source, see [For Developers](#for-developers) section below.
   # Bootstrap estimator wrapper
   boot_est = BootstrapEstimator(est, n_bootstrap_samples=10)
   boot_est.fit(Y, T, X, W)
-  treatment_effect_interval = boot_est.const_marginal_effect_interval(X_test, lower=1, upper=99)
+  treatment_effect_interval = boot_est.effect_interval(X_test, lower=1, upper=99)
```
Comment (kbattocchi) [inline README.md]: Once #76 is approved (ahem) this example should be further revised since explicitly using a wrapper will no longer be necessary.

Comment (moprescu) [reply README.md]: Sounds good! I will take a look/ approve #76 first and then update this PR accordingly.

File: README.md [hunk 2]
```diff
@@ -218,9 +220,13 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
...
 # References
 
+V. Syrgkanis, V. Lei, M. Oprescu, M. Hei, K. Battocchi, G. Lewis.
+**Machine Learning Estimation of Heterogeneous Treatment Effects with Instruments**
+[*ArXiv preprint arXiv:1905.10176*](https://arxiv.org/abs/1905.10176)
+
 M. Oprescu, V. Syrgkanis and Z. S. Wu.
 **Orthogonal Random Forest for Causal Inference.**
-[*ArXiv preprint arXiv:1806.03467*](http://arxiv.org/abs/1806.03467), 2018.
+[*Proceedings of the 36th International Conference on Machine Learning*](http://proceedings.mlr.press/v97/oprescu19a.html), 2019.
```
[the remark this answers is not in the document]
Comment (moprescu) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (kbattocchi) [reply README.md]: I just merged #76 so you can make this change now.

**Pull request merged**